src/PdoStreamIterator.php

```diff
@@ -78,7 +76,6 @@ public function __construct(
         PDO $connection,
         PDOStatement $statement,
         MessageFactory $messageFactory,
-        array $sql,
         ?int $batchSize,
         int $fromNumber,
         int $count,
```

Comment:
Is still needed here: https://github.com/oqq/pdo-event-store/blob/82a55d82a6bf93f20ecc0941198a608c22320590/src/PdoStreamIterator.php#L139